lib/entry.js

```diff
@@ -209,4 +136,74 @@ Ep.runModuleSetters = function (module) {
   }
 };
 
+function call(setter, name, value, callback) {
+  if (name === "__esModule") {
+    // Ignore setters asking for module.exports.__esModule.
+    return;
+  }
+
+  if (value !== value) {
+    value = NAN;
+  } else if (typeof value === "undefined") {
+    value = UNDEFINED;
+  }
+
+  setter.last = setter.last || Object.create(null);
+
+  if (setter.last[name] !== value) {
```

Comment:
Ok cool., it was done in the [callback before](https://github.com/benjamn/reify/blob/b5f78d92968562a8497beebb61e8e0417693d10a/lib/entry.js#L184-L188) (so I just carried it over).